web/survey/views.py

```diff
@@ -433,33 +434,23 @@ def manual_inspection_detection_view(request):
     # Handle GET request
     if request.method == 'GET':
         run_id = request.GET.get('run_id', None)
-        if not run_id:
+        if not int(run_id):
```

Comment:
What happens if `run_id` is `None`?

Reply:
This should be fixed now too with a try catch

```
try:
    run_id = int(run_id)
except ValueError:
    return HttpResponse('Run id is not an integer.', status=400)
```